### PR TITLE
feat(pci-public-ip): init react app

### PR DESCRIPTION
**Pull request by kqesar**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/network-public-ips`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | DTCORE-1964
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description
Initialize react application

### CODE REVIEW

File: packages/manager/apps/pci-public-ip/jest.config.cjs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const path = require('path');
```
Comment (frenautvh) [inline packages/manager/apps/pci-public-ip/jest.config.cjs:1]: file is not used

File: packages/manager/apps/pci-public-ip/tailwind.config.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: ['./src/**/*.{js,jsx,ts,tsx}'],
+  theme: {},
+  plugins: [],
```
Comment (frenautvh) [inline packages/manager/apps/pci-public-ip/tailwind.config.js:5]: tailwind config should be updated to disable preflight and include manager-components config (see ssh-key)

File: packages/manager/apps/pci-public-ip/package.json [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/manager-pci-public-ip-app' --include-dependencies -- npm run dev:watch --if-present",
+    "test": "vitest run"
+  },
+  "dependencies": {
+    "@ovh-ux/manager-config": "^7.3.0",
+    "@ovh-ux/manager-core-api": "^0.4.1",
+    "@ovh-ux/manager-react-shell-client": "^0.3.1",
+    "@ovh-ux/shell": "^3.2.0",
+    "@ovhcloud/manager-components": "^1.3.0",
+    "@ovhcloud/ods-common-core": "^17.2.0",
```
Comment (frenautvh) [inline packages/manager/apps/pci-public-ip/package.json:22]: 17.2.1

File: packages/manager/apps/pci-public-ip/src/components/BreadCrumbs.tsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  return (
+    <>
+      <ol>
+        {crumbs.map((crumb, index) => (
+          <li key={index}>{crumb}</li>
+        ))}
+      </ol>
+    </>
+  );
+}
```
Comment (seven-amid) [inline packages/manager/apps/pci-public-ip/src/components/BreadCrumbs.tsx:26]: why not the breadcrumb of Ods ?

**Review by sidlynx — APPROVED** [review]

**Pull request merged**